proposals/NNNN-library-evolution.md

```diff
@@ -0,0 +1,374 @@
+# Library Evolution for Stable ABIs
+
+* Proposal: [SE-NNNN](NNNN-non-exhaustive-enums.md)
```

Comment:
Needs basename fix, but I guess that's okay.

Reply:
oops